src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoNamingHelper.java

```diff
@@ -0,0 +1,75 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import org.sagebionetworks.bridge.config.Config;
+import org.sagebionetworks.bridge.config.Environment;
+
+/**
+ * Created by liujoshua on 4/18/16.
+ */
+public class DynamoNamingHelper {
```

Comment:
I'll declare it a bean in BridgeSpringConfig (and handle Config there)

I don't think we make anything a bean in bridge-base